.github/workflows/codeql-analysis.yml

```diff
@@ -1,20 +1,12 @@
-# For most projects, this workflow file will not need changing; you simply need
-# to commit it to your repository.
-#
-# You may wish to alter this file to override the set of languages analyzed,
-# or to provide custom queries or build logic.
 name: "CodeQL"
 
 on:
   push:
     branches: [master]
   pull_request:
-    # The branches below must be a subset of the branches above
     branches: [master]
   schedule:
-    - cron: '0 13 * * 6'
-
-permissions: {}
```

Comment:
Ditto. Let's keep the permissions as restricted as possible

Reply:
Done